oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -57,6 +58,9 @@
     public static final String NUMBER_OF_REPLICAS = "numberOfReplicas";
     public static final int NUMBER_OF_REPLICAS_DEFAULT = 1;
 
+    public static final String QUERY_FETCH_SIZES = "queryFetchSizes";
+    public static final Long[] QUERY_FETCH_SIZES_DEFAULT = new Long[]{100L, 1000L};
```

Comment:
Nulls are not needed in my view, so I would use "long[]" instead of "Long[]". Actually, because you later need to convert to integer, maybe just use "int[]" instead

Reply:
I need to keep `Long` here to match the type in `org.apache.jackrabbit.oak.api.Type<T>` since there are no types with subtype Int. I have converted `queryFetchSizes ` to `int[]`.

Reply:
+1